modules/default/clock/clock.js

```diff
@@ -196,6 +196,12 @@ Module.register("clock", {
 			}
 			const untilNextEvent = moment.duration(moment(nextEvent).diff(now));
 			const untilNextEventString = `${untilNextEvent.hours()}h ${untilNextEvent.minutes()}m`;
+
+			// Convert sunrise/sunset time to local representation of requested timezone
+			if (this.config.timezone) {
+				sunTimes.sunrise = sunTimes.sunrise.toLocaleString("en-us", { timeZone: this.config.timezone });
```

Comment:
probably needs to respect the "locale" config value instead of the hardcoded "en-us"